src/main/java/org/humancellatlas/ingest/schemas/Schema.java

```diff
@@ -0,0 +1,31 @@
+package org.humancellatlas.ingest.schemas;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.humancellatlas.ingest.core.AbstractEntity;
+
+/**
+ * Created by rolando on 18/04/2018.
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Schema extends AbstractEntity {
+    @JsonProperty("high-level-entity")
```

Comment:
do we have a convention for field names and relationship names? just noticed that some relationship names (even the existing ones) are in kebab-case and some are in camelCase

Reply:
Decided to make these kebab-case because the schemas themselves will contain kebab case. Just after realising that they actually use snake case. I'll revert these back to camel case though, for consistency. Can always change this later if requested